apps/ima/translations/src/en/fields.json

```diff
@@ -17,5 +17,47 @@
         "hint": "You cannot give immigration advice. You must answer all the questions as if you are the person named on the duty to remove letter."
       }
     }
+  },
+  "does-exception-apply":{
+    "legend": " ",
+    "options": {
+      "yes": {
+        "label": "Yes"
+      },
+      "no": {
+        "label": "No"
+      }
+    }
+  },
+  "does-exception-apply-detail": {
+    "label": "Explain why"
+  },
+  "is-life-threatened": {
+    "legend": " ",
+    "options": {
+      "yes": {
+        "label": "Yes"
+      },
+      "no": {
+        "label": "No"
+      }
+    }
+  },
+  "life-threatened-detail": {
+    "label": " "
+  },
+  "permission-to-enter-or-stay": {
+    "legend": " ",
```

Comment:
why is this empty? If is you make `isPageHeading` true for this field, you can set the legend here and remove the header from pages, json.